recipes/gprofiler/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "gprofiler" %}
```

Comment:
```suggestion
```
Please remove this jinja variable and replace all instances of `{{ name }}` with the actual package name.  This name will never change so there is no need for it to be a variable, unlike `python_min` and `version`, which will change over time.